pylintrc

```diff
@@ -64,12 +64,12 @@ bad-functions = map,filter,apply,input
 module-rgx = (([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+))$
 const-rgx = (([A-Z_][A-Z0-9_]*)|(__.*__)|log|urlpatterns)$
 class-rgx = [A-Z_][a-zA-Z0-9]+$
-function-rgx = ([a-z_][a-z0-9_]{2,30}|test_[a-z0-9_]+)$
+function-rgx = ([a-z_][a-z0-9_]{2,40}|test_[a-z0-9_]+)$
 method-rgx = ([a-z_][a-z0-9_]{2,40}|setUp|set[Uu]pClass|tearDown|tear[Dd]ownClass|assert[A-Z]\w*|maxDiff|test_[a-z0-9_]+)$
```

Comment:
@nedbat Who watches over this?  In a recent PR, @cahrens had to come up with the name display_name_w_default_escaped for us because display_name_with_default_escaped was too long.  I think poor names are worse than long names, so I am all for this change.